synapse/lib/autodoc.py

```diff
@@ -278,11 +319,58 @@ def getReturnLines(rtype, known_types=None, types_prefix=None, suffix=None, isst
         lines.append(line)
     return lines
 
+def runtimeGetReturnLines(rtype, isstor=False):
+    # Allow someone to plumb in name=Yields as a return type.
+    lines = ['']
+    whitespace = '   '
+    if isinstance(rtype, str):
+        lines.append('Returns:')
+        lines.append(f'    The type is {rtype}.')
+    elif isinstance(rtype, (list, tuple)):
+        assert len(rtype) > 1
+        tdata = ', '.join(rtype)
+        lines.append('Returns:')
+        lines.append(f'    The type may be one of the following: {tdata}.')
```

Comment:
This is schema valid but we don't have anything which uses it implemented, so it is not covered in tests.